src/rocksdb.rs

```diff
@@ -2434,7 +2433,7 @@ impl Env {
         let env = unsafe {
             crocksdb_ffi::crocksdb_ctr_encrypted_env_create(
                 base_env.inner,
-                mem::transmute(&ciphertext[0]),
+                ciphertext.as_ptr() as *const u8 as *const c_char,
```

Comment:
It should be fine to skip the middle `*const u8` cast.

```suggestion
                ciphertext.as_ptr() as *const c_char,
```